aten/src/ATen/native/Pow.cpp

```diff
@@ -38,12 +38,7 @@ DEFINE_DISPATCH(pow_tensor_tensor_stub);
 DEFINE_DISPATCH(pow_tensor_scalar_stub);
 
 TORCH_IMPL_FUNC(pow_Tensor_Tensor_out) (const Tensor& base, const Tensor& exp, const Tensor& out) {
-  if (exp.dim() == 0 && exp.device().is_cpu() && base.is_cuda()) {
-    // NOLINTNEXTLINE(cppcoreguidelines-pro-type-const-cast)
-    at::pow_out(const_cast<Tensor&>(out), base, exp.item()); // redispatch!
-  } else {
-    pow_tensor_tensor_stub(device_type(), *this);
-  }
+  pow_tensor_tensor_stub(device_type(), *this);
```

Comment:
does this require any changes on the cpu side, or cpu tensor_tensor_kernel would handle all the situations efficiently?

Reply:
The `if` condition checks `base.is_cuda()`, so this doesn't effect CPU.

Reply:
Oh I'm blind thank you